.travis.yml

```diff
@@ -15,4 +15,11 @@
 language: python
 python:
   - "3.5"
-script: nosetests -w lib/
+install:
+  - pip install flake8
+  - pip install ansible-lint
+script:
+  - nosetests -w lib/
+  - ansible-lint -x 204,301
```

Comment:
Why were these two excluded?  Not opposed to excluding them, just curious.  I found the following list of default rules while looking into this.

https://docs.ansible.com/ansible-lint/rules/default_rules.html

Reply:
Thanks Keith.
* I excluded 204 for now given the risk / reward consideration in refactoring the few longer-than-160-chars lines in Ansible playbooks. I will add a comment in the CI script to note that this is a future TODO.
* 301 is more interesting. It is flagged for specific cases where we set a fact based on a command output, without a when condition or creates condition. Such actions are necessarily repeated always by Ansible and hence are flagged. Again, I will flag this for a future TODO.

Reply:
Thanks for the information

Reply:
Created #341 to track and eventually fix these.